### PR TITLE
fix: sync columns before generate all in new doc editor

### PR DESCRIPTION
**Pull request by saravmajestic**
## Overview
Fix for: https://altimateai.atlassian.net/browse/AI-1514
- When user requests to generate docs for all columns, sync with db first and then generate with new columns
- also fixed minor css issue to show proper color for input/textarea placeholder text

## Checklist

- [x] I have run this code and it appears to resolve the stated issue
- [ ] `README.md` updated and added information about my change

### CODE REVIEW

**Review by suryaiyer95 — COMMENTED** [review]
Tested locally. @mdesmet @AdiGajbhiye can you review the code

File: webview_panels/src/modules/documentationEditor/components/docGenerator/BulkGenerateButton.tsx [hunk 1]
```diff
@@ -55,10 +55,17 @@ const BulkGenerateButton = () => {
...
+        "fetchMetadataFromDatabase",
+        {},
+      )) as { columns: DBTDocumentationColumn[] };
+
+      return await bulkGenerateDocs(columns);
+    } catch (err) {
+      panelLogger.error("Unable to generate docs for all columns");
     }
-    return bulkGenerateDocs(currentDocsData.columns);
+    return null;
```
Comment (mdesmet) [inline webview_panels/src/modules/documentationEditor/components/docGenerator/BulkGenerateButton.tsx:68]: Can't we just omit return null?

**Review by mdesmet — COMMENTED** [review]
LGTM except the return null

File: webview_panels/src/modules/documentationEditor/components/docGenerator/BulkGenerateButton.tsx (hunk 1 above)
Comment (saravmajestic) [reply webview_panels/src/modules/documentationEditor/components/docGenerator/BulkGenerateButton.tsx]: @mdesmet removed the return condition.

**Pull request merged**